staging/knative/charts/serving/crds/crds.yaml

```diff
@@ -26,7 +27,6 @@ kind: CustomResourceDefinition
 metadata:
   name: certificates.networking.internal.knative.dev
   labels:
-    serving.knative.dev/release: {{ .Chart.AppVersion | quote }}
```

Comment:
What is the reason behind removing this label?

Reply:
I was getting the following error during the CRD installation steep with Helm 3:
```
Error: failed to install CRD crds/crds.yaml: error parsing : error converting YAML to JSON: yaml: invalid map key: map[interface {}]interface {}{".Chart.AppVersion | quote":interface {}(nil)}
```
According to [Helm docs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you):
> There is now a special directory called crds that you can create in your chart to hold your CRDs. These CRDs are not templated, but will be installed by default when running a helm install for the chart.

As these labels are not used anywhere in KNative code, I've just removed them to resolve the templating issue.